src/index.js

```diff
@@ -1,65 +1,100 @@
 import './styles/main.scss';
 
-const dataBase = [
+const dataBaseDefault = [
   {
-    checked: true,
+    check: true,
     task: 'stop Procrastination',
     id: new Date('December 17, 1995 03:24:00'),
   },
   {
-    checked: false,
+    check: false,
     task: 'stop Procrastination',
     id: new Date('December 17, 1995 03:24:00'),
   },
   {
-    checked: true,
+    check: true,
     task: 'stop Procrastination',
     id: new Date('December 17, 1995 03:24:00'),
   },
 ];
 
```

Comment:
- [ ] As stated in the project requirement, Kindly all hardcoded items from the tasks array. Also kindly index your activities as stated in the project requirement. I see you used a date-time format to index your items

![image](https://user-images.githubusercontent.com/98453979/214881002-b9390706-8828-43ec-8c64-40f7a19c9ad6.png)

### Checkout requirement
![image](https://user-images.githubusercontent.com/98453979/214881277-10a0ba52-446e-4a81-95cd-4e095d723c3d.png)

- [ ] Kindly ensure that after editing a task, the changes are saved in the local storage. This would ensure that when the page is refreshed, the changes persist.